validator/testcases/regex.py

```diff
@@ -537,15 +537,15 @@ class Gecko53RegexTests(CompatRegexTestHelper):
     def tests(self):
         yield self.get_test_bug(
             1321556,
-            r'\burlbarBindings.xml\#splitmenu\b',
+            r'\burlbarBindings\.xml#splitmenu\b',
             'The splitmenu element has been removed.',
             'The splitmenu element has been removed.',
             log_function=self.err.warning,
             compat_type='error')
 
         yield self.get_test_bug(
             1331296,
-            r'\b[^-]*-moz-calc\b',
+            r'-moz-calc\b',
```

Comment:
This could cause some false-positives. We could potentially narrow it down to `\b[^\w]+-moz-calc\b` but I assume it's as slow as the previous one.